{"name": "test_addgroup_failure_003", "status": "failed", "statusDetails": {"message": "AssertionError: 405 != 500", "trace": "self = <test_Addgroup.TestAddgroup testMethod=test_addgroup_failure_003>\n\n        def test_addgroup_failure_003(self):\n            response = requests.get('http://127.0.0.1:8000/home/api/groups/addgroup/', params={\n      \"name\": \"Python 学习小组\",\n      \"created_by\": 1\n    })\n>           self.assertEqual(response.status_code, 500)\nE           AssertionError: 405 != 500\n\ntestplatform\\tests\\unit\\test_Addgroup.py:45: AssertionError"}, "start": 1743341764946, "stop": 1743341764952, "uuid": "e8b6a842-5329-4ea3-924d-d2da53390b99", "historyId": "2e59556b16c4b411a4c4dfbd07b7335c", "testCaseId": "2e59556b16c4b411a4c4dfbd07b7335c", "fullName": "testplatform.tests.unit.test_Addgroup.TestAddgroup#test_addgroup_failure_003", "labels": [{"name": "parentSuite", "value": "testplatform.tests.unit"}, {"name": "suite", "value": "test_Addgroup"}, {"name": "subSuite", "value": "TestAddgroup"}, {"name": "host", "value": "HO"}, {"name": "thread", "value": "17648-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testplatform.tests.unit.test_Addgroup"}]}